.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           name: clippy
           command: |
             cargo clippy --version
-            cargo clippy --all-features
+            cargo clippy --all-targets --all-features -- -D clippy::option_unwrap_used -D clippy::option_expect_used -D clippy::result_unwrap_used -D clippy::result_expect_used
```

Comment:
Maybe this check shouldn't be enabled until all the listed concerns above are solved.

Reply:
well, if we add this and the PR does not pass CI, then we may need to disable this check until all related issues are fixed

Reply:
After the new commits, two thing are holding back CI enforcement: `expect` usage in the FFI crate and `unwrap`s in the unused functions.

I think dealing with the ffi stuff won't be trivial

Reply:
Could you create an issue too track this? 
It would be nice to address the FFI issues and add this later on ` -- -D clippy::option_unwrap_used -D clippy::option_expect_used -D clippy::result_unwrap_used -D clippy::result_expect_used` 
We can later prioritize this task in the foollowing week.

Reply:
Done in https://github.com/Zondax/filecoin-rs/issues/146